pkg/image/apiserver/admission/imagepolicy/imagepolicy.go

```diff
@@ -39,12 +39,15 @@ func Register(plugins *admission.Plugins) {
 			if err != nil {
 				return nil, err
 			}
+			var config *imagepolicy.ImagePolicyConfig
```

Comment:
@bparees This seems ok, but does it work in the absence of internal and external registry hostnames?